File: module22/module01/module43/src/main/java/com/android/example/module22_module01_module43/ClassAAG.java
package com.android.example.module22_module01_module43;

public class ClassAAG {
	private 	com.android.example.module01_module39.ClassAAA instance_var_1_0 = new com.android.example.module01_module39.ClassAAA();
	private 	com.android.example.module01_module39.ClassAAJ instance_var_1_1 = new com.android.example.module01_module39.ClassAAJ();
	private 	com.android.example.module01_module39.ClassAAC instance_var_1_2 = new com.android.example.module01_module39.ClassAAC();
	private 	com.android.example.module01_module39.ClassAAC instance_var_1_3 = new com.android.example.module01_module39.ClassAAC();
	private 	com.android.example.module01_module39.ClassAAH instance_var_1_4 = new com.android.example.module01_module39.ClassAAH();

	public void method0(
		com.android.example.module01_module39.ClassAAJ param0,
		com.android.example.module01_module39.ClassAAF param1) throws Throwable {
		param0.method0(new io.reactivex.internal.disposables.ListCompositeDisposable(), new dagger.internal.DelegateFactory(), new io.reactivex.disposables.SerialDisposable(), new io.reactivex.subscribers.TestSubscriber());

		for (int iAb = 0; iAb < 0; iAb++) {
			io.reactivex.observers.TestObserver<Object> local_var_3_0 = new io.reactivex.observers.TestObserver();
			local_var_3_0.isDisposed();

			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_1 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_1.hasSubscription();

			io.reactivex.subscribers.TestSubscriber<Object> local_var_3_2 = new io.reactivex.subscribers.TestSubscriber();
			local_var_3_2.hasSubscription();

			local_var_3_0.onError(new java.lang.Throwable());

		}
		io.reactivex.observers.TestObserver<Object> local_var_2_2 = new io.reactivex.observers.TestObserver();
		local_var_2_2.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.assertNotSubscribed();

	}

	public void method1(
		com.android.example.module01_module39.ClassAAF param0,
		com.android.example.module01_module39.ClassAAF param1) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.isCancelled();

	}

	public void method2(
		com.android.example.module01_module39.ClassAAI param0,
		com.android.example.module01_module39.ClassAAE param1,
		com.android.example.module01_module39.ClassAAF param2) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.isDisposed();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.isDisposed();

	}

	public void method3(
		com.android.example.module01_module39.ClassAAF param0,
		com.android.example.module01_module39.ClassAAJ param1) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.requestMore(486274779l);

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_3 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_3.hasSubscription();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.isDisposed();

	}
}

File: module07/module18/module3/src/main/java/com/android/example/module07_module18_module3/ClassAAG.kt
package com.android.example.module07_module18_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAG {
	private 	val instance_var_1_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
	private 	val instance_var_1_1: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
	private 	val instance_var_1_2: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
	private 	val instance_var_1_3: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
	private 	val instance_var_1_4: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()

	fun method0(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param2: dagger.internal.DelegateFactory<kotlin.Any>,
		param3: dagger.internal.DelegateFactory<kotlin.Any>
	) {
		val local_var_2_4: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
		local_var_2_4.get()

		for (iAb in 0..1) {
			param0.isDisposed()

			val local_var_3_0: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
			local_var_3_0.get()

		}
		param0.assertSubscribed()

	}

	fun method1(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		val local_var_2_2: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
		local_var_2_2.get()

		param0.isCancelled()

	}

	fun method2(
		param0: dagger.internal.DelegateFactory<kotlin.Any>
	) {
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.cancel()

		param0.get()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.isCancelled()

		val local_var_2_3: dagger.internal.DelegateFactory<kotlin.Any> = dagger.internal.DelegateFactory()
		local_var_2_3.get()

	}

	fun method3(
		param0: dagger.internal.DelegateFactory<kotlin.Any>,
		param1: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		param0.get()

		param0.get()

	}
}
